packages/message-db/message-db/src/lib/Category.ts

```diff
@@ -194,6 +194,8 @@ class InternalCategory<Event, State, Context>
   private async loadAlgorithm(
     streamId: Equinox.StreamId,
     requireLeader: boolean,
+    initial: State,
+    version = 0n,
```

Comment:
not sure about defalt arg as you didnt do it for loadbatched in the other impl

Reply:
fair, went through and reduced the line count through "uglification" as well